mapmatcher/src/main/java/io/github/nik9000/mapmatcher/ListMatcher.java

```diff
@@ -41,7 +41,19 @@ public static ListMatcher matchesList() {
   public static ListMatcher matchesList(List<?> list) {
     ListMatcher matcher = matchesList();
     for (Object item : list) {
-      matcher = matcher.item(item);
+      matcher = matcher.item(item, false);
+    }
+    return matcher;
+  }
+
+  /*
```

Comment:
I think this should be javadoc like `/**` instead of `/*` just so when folks mouseover the method it'll show nicely.